servers/quarkus-tests/src/main/java/org/projectnessie/quarkus/tests/profiles/PostgresTestResourceLifecycleManager.java

```diff
@@ -38,7 +38,7 @@ public void setIntegrationTestContext(DevServicesContext context) {
 
   @Override
   public Map<String, String> start() {
-    String version = System.getProperty("it.nessie.container.postgres.tag", "14");
+    String version = System.getProperty("it.nessie.container.postgres.tag");
```

Comment:
`version` could never be `null` before - now it can - it should throw an exception with a meaningful message.

Reply:
Agree